Description: Work with the older python-calendar in Debian
 Revert code to use pycalendard.calendar which was used by upstream
 before version 6.0.
Author: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2016-02-02
License: Apache 2 (same as caldav-tester itself)

Index: caldav-tester-packaging/src/caldavtest.py
===================================================================
--- caldav-tester-packaging.orig/src/caldavtest.py	2016-02-02 09:21:14.598371067 +0000
+++ caldav-tester-packaging/src/caldavtest.py	2016-02-02 09:22:31.946474450 +0000
@@ -19,7 +19,7 @@
 """
 
 from cStringIO import StringIO
-from pycalendar.icalendar.calendar import Calendar
+from pycalendar.calendar import PyCalendar
 from src.httpshandler import SmartHTTPConnection
 from src.jsonPointer import JSONMatcher
 from src.manager import manager
@@ -1015,7 +1015,7 @@
     def _calProperty(self, propertyname, respdata):
 
         try:
-            cal = Calendar.parseText(respdata)
+            cal = PyCalendar.parseText(respdata)
         except Exception:
             return None
 
